### ISSUE TITLE
As a user, I should be able to rely on some accessibility features and elements when using the app

### ISSUE THREAD

**Issue by SheIsBukki** [SheIsBukki/charbage#26]
- [ ] Implement and provide accessible features and elements users can rely on when using the app, such as tooltips that inform them what an icon does when they hover over it

I will fill up more as I remember...
